README.md

```diff
@@ -0,0 +1,62 @@
+# Store and process the Crossref Database
+
+This repository downloads Crossref metadata using the [Crossref API](https://github.com/CrossRef/rest-api-doc/blob/master/rest_api.md).
+The items retrieved are stored in MongoDB to preserve their raw structure.
+This design allows for flexible downstream analyses.
+
+## MongoDB
+
+MongoDB is run via [Docker](https://hub.docker.com/_/mongo/).
+It's available on the host machine at http://localhost:27017/.
+
+```sh
+docker run \
+  --name=mongo-crossref \
+  --publish=27017:27017 \
+  --volume=`pwd`/mongo.db:/data/db \
+  --rm \
+  mongo:3.4.2
+```
+
+## Execution
+
+With mongo running, execute with the following commands:
+
+```sh
+# Download all works
+# To start fresh, use `--cursor=*`
```

Comment:
Added more doc in https://github.com/greenelab/crossref/pull/2/commits/d2213939e507ed007d2cffe6202bead839b1ecb6